Custom-Audio-Driver/External-Audio-Device/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -2,68 +2,93 @@
   "images" : [
     {
       "idiom" : "iphone",
-      "size" : "29x29",
```

Comment:
Here seems there are no changes, just the order of the json content. If that is the case, please remove it from the PR.

Reply:
the resolutions are different

Reply:
Which ones? I see a few new ones, but nothing is changed or removed. What Xcode version did you use?

Reply:
oh wait yes I see what you mean now. Will remove 👍